admin/services/check-start.service.js

```diff
@@ -21,6 +21,10 @@ const checkStartService = {
     const checkForm = await checkFormService.allocateCheckForm()
     const checkWindow = await checkWindowDataService.sqlFindOneCurrent()
 
+    if (checkWindow && moment(checkWindow.checkStartDate).isAfter(moment.utc())) {
```

Comment:
This will return a 500 server error.  Isn't logging in too early a client error and should result in a 4xx code?  Or perhaps server temporarily unavailable `503`?

We also need to check that the check-window has not ended. this is also an error, but a more permanent one.

I'm thinking that this check needs it's own service and service call.